turbine/src/sigverify_shreds.rs

```diff
@@ -56,11 +60,12 @@ pub fn spawn_shred_sigverify(
             if deduper.maybe_reset(&mut rng, DEDUPER_FALSE_POSITIVE_RATE, DEDUPER_RESET_CYCLE) {
                 stats.num_deduper_saturations += 1;
             }
+            // We can't store the keypair outside the loop
+            // because the identity might be hot swapped.
+            let keypair: Arc<Keypair> = cluster_info.keypair().clone();
```

Comment:
Can type not be inferred here?

Reply:
I think it can be inferred fine and it is redundant. But the function below only requires a `&Keypair` and not `Arc<Keypair>` and I just wanted to explicitly avoid
```rust
let keypair: &Keypair = &cluster_info.keyapir();
```
which _unfortunately_ compiles but holds the lock on `ClusterInfor.keypair`.